docs/robots/warthog/user_manual_warthog.mdx

```diff
@@ -92,30 +151,55 @@ For more information and guidance on mounting payload structures on top of Warth
 | Suspension                      | Geometric Passive Articulation                                                             |
 | Drive Configuration             | 4 X 4 Skid Steer                                                                           |
 | Operating Environment           | Outdoor                                                                                    |
-| Traction                        | Ø610 mm (24") Argo tire (Turf tire or 300 mm (12") wide Quad Track System optional)        |
-| Battery Chemistry               | AGM sealed lead-acid (lithium optional)                                                     |
-| Capacity                        | 105 Ah at 48 V, expandable to 200 Ah with lithium option                                    |
-| Nominal Run Time                | Lead-acid: 2.5 hrs, lithium: 6 hrs                                                          |
+| Traction                        | Ø610 mm (24") Argo tire (Turf tire or 300 mm (12") wide Quad Track System (optional)       |
```

Comment:
Parenthesis is still open  (  ).
Could change this to italics.

Reply:
Updated